css-canvas-medical/styles.css

```diff
@@ -0,0 +1,239 @@
+.container {
+  height: 120rem;
```

Comment:
don't use rem for the height of a container, and your container doesn't need a height, its content is more than the size of the page so it will fill up naturally